tests/arp/test_stress_arp.py

```diff
@@ -215,3 +218,62 @@ def test_ipv6_nd(duthost, ptfhost, config_facts, tbinfo, ip_and_intf_info,
                     raise e
             # Wait for 10 seconds before starting next loop
             time.sleep(10)
+
+
+def add_nd_nonstop(ptfadapter, ip_and_intf_info, ptf_intf_index, nd_available, stop_event):
```

Comment:
Could you add a description for this function?

Reply:
Added